JHUGenerator/mod_VHiggs.F90

```diff
@@ -481,13 +481,13 @@ complex(8) function MATRIXELEMENT0(MomExt,mass,helicity,id,useA)
 
       if(.not.(useA(1) .and. abs(id(1)).eq.convertLHE(Pho_))) then
          current1 = -current1 + scrc(MomExt(:,3),current1)/q3_q3
-         if(UseVprime) then
+         if(includeVprime) then
```

Comment:
Ulascan please check that this is what you meant.  I'm pretty certain that UseVprime (or Zprime) here is wrong.

Reply:
No, it is a term related to propagator, so UseZ/Wprime is correct.
The Z/W propagator is (-gmunu+kmu*knu/k^2)/[regular term]. This term allows for MASSIVE currents if one prefers to use them in the future and has zero contribution if current is massless (k.J=0 if J is massless), not something we had for the HVV process. Please revert this change.